src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -20,16 +16,30 @@
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.team2412.robot.Hardware;
+import frc.team2412.robot.Robot;
 import frc.team2412.robot.sim.PhysicsSim;
 import frc.team2412.robot.util.ModuleUtil;
 import frc.team2412.robot.util.PFFController;
+import frc.team2412.robot.util.gyroscope.Gyroscope;
+import frc.team2412.robot.util.gyroscope.NavXGyro;
+import frc.team2412.robot.util.gyroscope.Pigeon2Gyro;
+import frc.team2412.robot.util.motorcontroller.BrushlessSparkMaxController;
+import frc.team2412.robot.util.motorcontroller.MotorController;
+import frc.team2412.robot.util.motorcontroller.MotorController.MotorControlMode;
+import frc.team2412.robot.util.motorcontroller.MotorController.MotorNeutralMode;
+import frc.team2412.robot.util.motorcontroller.TalonFXController;
 
 public class DrivebaseSubsystem extends SubsystemBase {
 
-	private static final double ticksPerRotation = 2048.0; // for the talonfx
+	private static final boolean isComp = Robot.getInstance().isCompetition();
+
+	private static final double maxDriveSpeedMeters = 4.4196;
```

Comment:
Consider adding either an `isComp ?` and adding a value for the test drivebase, or updating the name to reflect this is just for comp.